packages/bpk-component-map/src/__snapshots__/BpkMapMarker-test.js.snap

```diff
@@ -24,7 +24,7 @@ exports[`BpkMapMarker should render correctly with a "arrowClassName" attribute
     <div
       className="bpk-map-marker__arrow custom-class-1 custom-class-2"
     >
-      <Component />
+      <_default />
```

Comment:
I'm struggling to find documentation for why this would change. It seems ok though so I'm not concerned. My gut feeling is that this is happening for components that don't have a name. In this case it's an imported icon.

Reply:
Yeah I would read that as "The default import from the package"